Update hybrid-picker-in-the-sharepoint-online-admin-center.md (#1053)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointServer/hybrid/hybrid-picker-in-the-sharepoint-online-admin-center.md

  
## How we use your data

> Hybrid Picker is part of the Office 365 Support Assistant. Hybrid Picker may use your information in the ways outlined by our Privacy Statement seen [here](https://privacy.microsoft.com/en-us/privacystatement) (click on the **How we use personal data** link ). 

Reviewer: ```suggestion
Hybrid Picker is part of the Office 365 Support Assistant. Hybrid Picker may use your information in the ways outlined in our [Privacy Statement](https://privacy.microsoft.com/privacystatement) (click on the **How we use personal data** link ). 
```

> Hybrid Picker is part of the Office 365 Support Assistant. Hybrid Picker may use your information in the ways outlined by our Privacy statement seen [here](https://privacy.microsoft.com/en-us/privacystatement) (please remember to click on 'How we use personal data' link ). You'll find a more detailed discussion of Microsoft's privacy policies, right [here](https://www.microsoft.com/EN-US/privacystatement/OnlineServices/Default.aspx) (please remember to click the 'Privacy Statement' link in the privacy policies article)! 

Reviewer 2: ```suggestion
Hybrid Picker is part of the Office 365 Support Assistant. Hybrid Picker may use your information in the ways outlined by our Privacy Statement seen [here](https://privacy.microsoft.com/en-us/privacystatement) (click on the **How we use personal data** link ). 
```
The 2 links go to the same page, and there is no 'Privacy Statement' link to click on.
